services/web/src/components/search/context.js

```diff
@@ -0,0 +1,2 @@
+import React from 'react';
```

Comment:
shouldnt this file be Context.js to align with how you you are importing it

Reply:
yeah fair enough